table/server/common/src/main/java/alluxio/table/common/udb/PathTranslator.java

```diff
@@ -27,7 +31,10 @@
 public class PathTranslator {
   private static final Logger LOG = LoggerFactory.getLogger(PathTranslator.class);
 
-  private BiMap<String, String> mPathMap;
+  private final BiMap<AlluxioURI, AlluxioURI> mPathMap;
+
+  @VisibleForTesting
+  static String sSchemeAuthorityPrefix = null;
```

Comment:
Can we try `private static final` (making it constant) and initialized this constant to `ConfigurationUtils.getSchemeAuthority(ServerConfiguration.global());`

In test (e.g., https://github.com/Alluxio/alluxio/pull/13708/files#diff-5c564844c705a2abf294409baa2f79a7bde047243560c17ee60c824405239a5bR159),
you can follow https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#managing-global-state to inject the prefix to be `alluxio://master`

Reply:
Done. PTAL.